var/spack/repos/builtin/packages/py-morph-tool/package.py

```diff
@@ -14,24 +14,17 @@ class PyMorphTool(PythonPackage):
     url      = "https://pypi.io/packages/source/m/morph-tool/morph-tool-0.2.5.tar.gz"
 
     version('develop', branch='master')
-    version('2.2.21', sha256='37d932b8b9561d9b5b4d0d3cdb1b146e9d4e3443526b8ef6e88aeeac36528c7e')
-    version('0.2.10', sha256='e46835b9c25532cbee070141ad9f7d3d47109d5473074b47aad08f51bdc40157')
-    version('0.2.5', sha256='7157039a7a421cfbdb75fd305975a6e6ce7667177dc68bd299645b62389622be')
-    version('0.2.3', sha256='767effaa4d2e8c7dfee878c6d48c6647f29b19fee6f790213783c04d8951fee3')
-    version('0.2.1', tag='morph-tool-v0.2.1', git='ssh://bbpcode.epfl.ch/nse/morph-tool')
-    version('0.1.14', tag='morph-tool-v0.1.14', git='ssh://bbpcode.epfl.ch/nse/morph-tool')
+    version('2.3.1', sha256='37d932b8b9561d9b5b4d0d3cdb1b146e9d4e3443526b8ef6e88aeeac36528c7e')
 
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('neuron+python', when='@0.2.1:', type='run')
-    depends_on('py-bluepyopt@1.9.37:', type='run', when='@0.2.1:')
-    depends_on('py-click@7.0:', type='run')
-    depends_on('py-functools32', when='@:0.2.3^python@:2.99', type='run')
+
+    depends_on('py-bluepyopt@1.9.37:', type='run')
+    depends_on('py-click@6.7:', type='run')
+    depends_on('py-dask[bag]@2.19.0:', type='run')
```

Comment:
pretty sure that this is not valid in Spack…
```suggestion
    depends_on('py-dask+bag@2.19.0:', type='run')
```

Reply:
Thanks. I think that's the bug I was looking for.